modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -52,6 +53,10 @@ func (a TransferAuthorization) Accept(goCtx context.Context, msg proto.Message)
 		return authz.AcceptResponse{}, errorsmod.Wrapf(ibcerrors.ErrNotFound, "requested port and channel allocation does not exist")
 	}
 
+	if !isAllowedForwarding(msgTransfer.Forwarding.Hops, a.Allocations[index].AllowedForwardingHops) {
+		return authz.AcceptResponse{}, errorsmod.Wrapf(ErrInvalidForwarding, "not allowed forwarding hops")
```

Comment:
think we just need

```suggestion
		return authz.AcceptResponse{}, errorsmod.Wrap(ErrInvalidForwarding, "not allowed forwarding hops")
```

(I think this also applies to the line above, no need for `Wrapf`)

Reply:
Yep, good point, fixed both!